CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 - Support color customization
 
+### Fixed
+
+- `sandstone/VideoPlayer` to return the current state of the media when handling event and not to invoke the mediaControl related callback when video source is unavailable
```

Comment:
Let's shorten this...

```suggestion
- `sandstone/VideoPlayer` to handle mediaControl related callbacks properly
```

Reply:
And Add a "added" changelog



- `sandstone/VideoPlayer` onWillPause, , ,,  , , props

Reply:
It's fixed.